docs/Installation.md

```diff
@@ -72,9 +74,20 @@ Zonemaster::CLI, see the [declaration of prerequisites].
    ```
 
 
-### Installation on Debian or Ubuntu
+### Installation on Debian
 
-1) Update configuration of "locale"
+Using pre-built packages is the preferred method for Debian. If you prefer to
+install from CPAN instead, follow the steps for Ubuntu.
+
+1) Configure Zonemaster repository
+   ```sh
+   curl -Ls https://package.zonemaster.net/setup.sh | sudo sh
+   ```
```

Comment:
I think that "Configure Zonemaster repository" is confusing. What is updated is a the local list of repositories, not a Zonemaster repository.

I do not think that anyone should execute the command curl -Ls https://package.zonemaster.net/setup.sh | sudo sh. This something unknown that you will execute as root. I do not think we should have that in our instructions. It is better have an separate section at the end (appendix) with the commands. Instead we should comment that using package will make upgrade easier.

Reply:
I can change it to:
Download install script
```
curl -Ls https://package.zonemaster.net/setup.sh
```
Review the content 
```
less setup.sh
```
Execute the install script
```
sudo sh setup.sh
````

Reply:
> Configure Zonemaster repository
I can change it to `Configure Zonemaster packages repository` if that is better

Reply:
>I do not think that anyone should execute the command

Otherwise the instruction would be (master branch only)

```
curl -Ls https://package.zonemaster.net/zonemaster-archive-keyring.gpg | sudo tee /usr/share/keyrings/zonemaster-archive-keyring.gpg > /dev/null

. /etc/os-release

cat <<EOF | sudo tee /etc/apt/sources.list.d/zonemaster.sources
Types: deb deb-src
URIs: https://package.zonemaster.net/debian/
Suites: ${VERSION_CODENAME}
Architectures: $(dpkg --print-architecture)
Components: main
Signed-By: /usr/share/keyrings/zonemaster-archive-keyring.gpg
EOF

sudo apt update
```

Reply:
I guess an alternative is `sudo dpkg -i package_file.deb` but the difference is minor, I guess, because then you trust that the deb package does not execute evil code.

Reply:
I honestly don't see much difference between downloading and installing a deb file or downloading and executing a sh script. I thought about that alternative too, but I don't think it is worth the extra work.